### PR TITLE
Add 100+ test to object origin mixin

### PR DESCRIPTION
**Pull request by asturur**
To be sure to do not mess up them in the future.

### CODE REVIEW

**Comment by asturur** [comment]
@kangax for me this is done. After pulling this we can pull also the #2257 if still mergeable.

File: test/unit/object_origin.js [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+    deepEqual(rect.top, 56.47152872702093);
+    equal(rect.originX, 'left');
+
+    p = rect.adjustPosition('center');
+    deepEqual(rect.left, 67.76608177155967);
+    deepEqual(rect.top, 67.94305745404185);
+    equal(rect.originX, 'center');
+
+    p = rect.adjustPosition('right');
+    deepEqual(rect.left, 84.1491226573395);
```
Comment (kangax) [inline test/unit/object_origin.js]: I'm a little unsure about such precise fractions. Usually they always cause problems. But maybe OK in this case.

Comment (asturur) [inline test/unit/object_origin.js]: i was thinking same.
first time that this create some error in tests we will add toFixed(NUM_FRACTION_DIGIT) to every coordinate  of every point of the test.

Comment (kangax) [inline test/unit/object_origin.js]: Alright, let's see how it goes

**Pull request merged**